include/z64animation.h

```diff
@@ -30,12 +30,18 @@ typedef enum AnimationMode {
     /* 5 */ ANIMMODE_LOOP_PARTIAL_INTERP
 } AnimationMode;
 
-typedef enum { 
+typedef enum {
     /* -1 */ ANIMTAPER_DECEL = -1,
-    /*  0 */ ANIMTAPER_NONE, 
+    /*  0 */ ANIMTAPER_NONE,
     /*  1 */ ANIMTAPER_ACCEL
 } AnimationTapers;
 
+typedef enum LevelOfDetail {
+    /* 0 */ LEVEL_OF_DETAIL_NEAR,
+    /* 1 */ LEVEL_OF_DETAIL_FAR,
+    /* 2 */ LEVEL_OF_DETAIL_MAX
+} LevelOfDetail;
```

Comment:
```suggestion
typedef enum LevelOfDetail {
    /* 0 */ LOD_NEAR,
    /* 1 */ LOD_FAR,
    /* 2 */ LOD_MAX
} LevelOfDetail;
```
Since this abbreviation is well established? Might even be worth naming `LevelOfDetail` -> `LOD`

Reply:
Is this another "programmers can't name things" thing? How can a level be near or far?

Reply:
"Level" is a different word and meaning to "Level-Of-Detail", not sure why you isolated "Level". The "Level of Detail" applied to a model when the camera is "near" the model or "far" from the model makes sense to me?

Reply:
No, that's a correlation/causation mistake. You almost always want a high level of detail when close and low when further away, but it doesn't necessarily have to be so.

Reply:
What does correlation/causation have to do with code logic? This is the function that reads the LOD enum, that's based on `projectedPos.z`, which is related to distance from camera. Here, the definition of near is `< 160.f`, and far is `>= 160.f`

```c
SkelAnime_DrawLod(play, this->arrow.skelAnime.skeleton, this->arrow.skelAnime.jointTable, NULL, NULL,
                          &this->actor,
                          this->actor.projectedPos.z < 160.0f ? LEVEL_OF_DETAIL_NEAR : LEVEL_OF_DETAIL_FAR);
```